### ISSUE TITLE
Build error with 1.1.0 and Svelte 5

### ISSUE THREAD

**Issue by Kadalia** [radiofrance/svelte-youtube-lite#4]
Hello,
I was able to use svelte-youtube-lite ```0.6.1``` in a SvelteKit application (no problem with Svelte 5, need to --force npm install)
But I encounter a build error with svelte-youtube-lite ```1.1.0```:
```
14.54 error during build:
14.54 [vite-plugin-svelte] [plugin vite-plugin-svelte] node_modules/svelte-youtube-lite/dist/Youtube.svelte (2:13): node_modules/svelte-youtube-lite/dist/Youtube.svelte:2:13 Unexpected token
14.54 file: /app/node_modules/svelte-youtube-lite/dist/Youtube.svelte:2:13
14.54 
14.54  1 |  <script>
14.54  2 |    import type { Snippet } from 'svelte';
14.54                     ^
14.54  3 |    import PlayButton from './PlayButton.svelte';
14.54  4 | 
```
By the way ... thank you for your work ! (I'm french ... Radio France rocks !!)
I see you are active recently working on Svelte 5 compatibility, work in progress probably, or am I missing something ?

Thank you,
Philippe

**Comment by romaindurand** [comment]
Thank you so much for your feedback and the nice comment !
Svelte 5 migration was done using the provided migration script. Since the demo page was working fine I thought it would also work as a lib but didn't bother to actually try it.
I published a `v1.2.1` that was remade from scratch using the new svelte CLI, tell me if it works for you

**Comment by Kadalia** [comment]
It works ! 
```v1.2.1``` builds with Svelte 5.
Thank you
